components/builder-api-client/src/builder.rs

```diff
@@ -531,6 +533,198 @@ impl BuilderAPIProvider for BuilderAPIClient {
             .ok_if(&[StatusCode::NO_CONTENT])
     }
 
+    /// Accepts an origin member invitation
+    ///
+    ///  # Builder API endpiont (api.raml permalink)
+    ///    * https://github.com/habitat-sh/builder/blob/da72e9fb86e24d9076268b6b1c913b7531c83ed9/components/builder-api/doc/api.raml#L883
+    ///
+    ///  # Arguments
+    ///    * Origin: &str - origin name where membership invitation exists
+    ///    * Token: &str - bearer token for authentication/authorization
+    ///    * Invitation Id: u64 - id of invitation to accept
+    ///
+    ///  # Expected API response on success
+    ///    * HTTP 204
+    ///
+    ///  # Return
+    ///    * Result<()>
+    fn accept_origin_invitation(&self,
+                                origin: &str,
+                                token: &str,
+                                invitation_id: u64)
+                                -> Result<()> {
+        debug!("Accepting invitation id {} in origin {}",
+               invitation_id, origin);
+
+        let path = format!("depot/origins/{}/invitations/{}", origin, invitation_id);
+
+        self.0
+            .put(&path)
+            .bearer_auth(token)
+            .send()?
+            .ok_if(&[StatusCode::NO_CONTENT])
+    }
+
+    /// Marks an origin member invitation ignored
+    ///
+    ///  After ignoring, the user will no longer see the invitation
+    ///
+    ///  # Builder API endpiont (api.raml permalink)
```

Comment:
You're a wonderful human being. Thanks for putting this in here.

Reply:
![](https://media.giphy.com/media/nj0k2dDx4YAgg/giphy.gif)